pkg/ebpf/c/tracer-ebpf.h

```diff
@@ -3,8 +3,8 @@
 
 #include <linux/types.h>
 
-static const __u64 TRACER_IPV6_DISABLED = 0;
-static const __u64 TRACER_IPV6_ENABLED = 1;
+static const __u64 ENABLED = 0;
+static const __u64 DISABLED = 1;
```

Comment:
You have switched the values of enabled and disabled here. Is that intentional?

Reply:
No, it is not intentional. Great catch! I will check why it was not being caught by any of the tests.

Reply:
🙇‍♂️  I was wondering the same thing about the tests.